.github/CODEOWNERS

```diff
@@ -7,6 +7,10 @@
 # These *code owners* will be the default owners for everything in the repository.
 * @chrisdholt @janechu @nicholasrice @EisenbergEffect
 
+# All *code owners* who aren't on the "everything" list must be have ownership of
+# the change directory in order to approve PR's that include change files
+/change/ @bheston @scomea @radium-v @kingoftac
```

Comment:
This will negate any other reviewers. Well likely want to add Fast Element owners here as well.

Reply:
Done.  I would have thought that folks with "*" approval would override.

Reply:
That should be true. If not true, then should line 8 be removed.

Reply:
Default owners are for anything not explicitly stated here, not for override. 8 is fine, this just needs to include all relevant owners which could see change files.